fatiando/seismic/tests/test_seismic_srtomo.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+from numpy.testing import assert_array_almost_equal, assert_allclose
+from pytest import raises
+
+from fatiando.seismic import srtomo
+from fatiando.mesher import Square, SquareMesh
+from fatiando.seismic import ttime2d
+
+
+def test_general():
+    """
+    General test of the class SRTomo, as it is in the docs of this class.
+    """
+    model = SquareMesh((0, 10, 0, 10), shape=(2, 1), props={'vp': [2., 5.]})
+    src = (5, 0)
+    srcs = [src, src]
+    recs = [(0, 0), (5, 10)]
+    ttimes = ttime2d.straight(model, 'vp', srcs, recs)
+    tomo = srtomo.SRTomo(ttimes, srcs, recs, model)
```

Comment:
Please create a `mesh` object using `SquareMesh` to use with the tomo class. The `model` object is mutable and you won't have it if you're not running a synthetic data test.